src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -528,6 +534,9 @@ class LicenceService(
           ),
         )
       }
+      inactivateInProgressLicenceVersions(
```

Comment:
some recursion here?
`inactivateInProgressLicenceVersions -> inactivateLicences  -> inactivateInProgressLicenceVersions`

Reply:
yes there is recursion but it stops quickly as in progress versions don't have any versions linked to them. It works ok when I tested locally. I did add an optional param to stop the recursion when I was looking at ways around this but took it out, I'll add it back

Reply:
if you think its fine, then thats' good enough!

Reply:
updated to avoid the recursion to be safe